js_modules/dagit/packages/core/src/workspace/VirtualizedAssetTable.tsx

```diff
@@ -0,0 +1,376 @@
+          <AssetLink path={path} url={linkUrl} isGroup={false} icon="asset" />
+          <div
+            style={{
+              maxWidth: '100%',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+            }}
+          >
+            <Caption
+              style={{
+                color: Colors.Gray500,
+                whiteSpace: 'nowrap',
+              }}
+            >
+              {asset?.definition?.description}
+            </Caption>
+          </div>
+        </RowCell>
+        <RowCell>
+          {liveData?.lastMaterialization ? (
+            <Box flex={{gap: 8, alignItems: 'center'}}>
+              <>
+                <AssetRunLink
+                  runId={liveData.lastMaterialization.runId}
+                  event={{
+                    stepKey: liveData.stepKey,
+                    timestamp: liveData.lastMaterialization.timestamp,
+                  }}
+                >
+                  <TimestampDisplay
+                    timestamp={Number(liveData.lastMaterialization.timestamp) / 1000}
+                    timeFormat={{showSeconds: false, showTimezone: false}}
+                  />
+                </AssetRunLink>
+              </>
+              <ComputeStatusNotice computeStatus={liveData?.computeStatus} />
+            </Box>
+          ) : (
+            <LoadingOrNone queryResult={queryResult} />
+          )}
+        </RowCell>
+        <RowCell>
+          {liveData ? (
+            <AssetLatestRunWithNotices liveData={liveData} />
+          ) : (
+            <LoadingOrNone queryResult={queryResult} />
+          )}
+        </RowCell>
+        <RowCell>
+          {asset ? (
+            <div>
+              <AssetActionMenu repoAddress={repoAddress} asset={asset} />
+            </div>
+          ) : null}
+        </RowCell>
+      </RowGrid>
+    </Row>
+  );
+};
+
+export const GroupNameRow: React.FC<{
+  repoAddress: RepoAddress;
+  groupName: string;
+  assetCount: number;
+  expanded: boolean;
+  height: number;
+  start: number;
+  onToggle: (groupName: string) => void;
+}> = ({repoAddress, groupName, assetCount, expanded, height, start, onToggle}) => {
+  return (
+    <Row $height={height} $start={start}>
+      <Box
+        background={Colors.Gray50}
+        flex={{direction: 'row', alignItems: 'center', gap: 8, justifyContent: 'space-between'}}
+        padding={{horizontal: 24}}
+        border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+        style={{height: '100%'}}
+      >
+        <Box flex={{alignItems: 'center', gap: 8}}>
+          <Icon name="asset_group" />
+          <ExpandButton onClick={() => onToggle(groupName)} $open={expanded}>
+            <Icon name="arrow_drop_down" size={20} />
+          </ExpandButton>
+          {groupName === UNGROUPED_NAME ? (
+            <div>Ungrouped assets</div>
+          ) : (
+            <>
+              <div>Asset group</div>
+              <strong>{groupName}</strong>
```

Comment:
I'm not sure we need the "Asset group" label before the group name here, I think the asset group icon is probably enough. Can always come back to this UI polish though.

Reply:
Yeah, I just wasn't sure if it was obvious enough that that's what these rows were, especially if they haven't been expanded.